### ISSUE TITLE
Exiting a vehicle triggers sneak mode

### ISSUE THREAD

**Issue by matejdro** [jrbudda/Vivecraft_116#94]
1. Enter minecart
2. Press sneak button (simulating shift) on the VR controller
3. You will exit the minecart BUT you will also start sneaking

I guess sneak mode should only start when player is not in a vehicle while pressing sneak button?

**Comment by Techjar** [comment]
I'm pretty sure this is vanilla behavior.

**Comment by matejdro** [comment]
Not really. With vanilla, you have to HOLD down the key to sneak, so if you sneak for half a second while exiting a vehicle, it does not cause any problems.

But with Vivecraft, sneaking is a toggle, which is way more annoying.

**Comment by Techjar** [comment]
You can make it not a toggle if you like, that's just our default bindings.

**Comment by matejdro** [comment]
I understand. But toggle is not a problem for me in general. What is a problem for me is "Toggle while in a vehicle", hence this issue report.

**Comment by Techjar** [comment]
The toggle is handled outside of Vivecraft code so there's not really any way to fix it.

**Comment by matejdro** [comment]
Maybe you could do a workaround and bind to the exit vehicle event and turn off sneak mode after exiting a vehicle?

**Comment by Techjar** [comment]
I don't want to change the vanilla behavior for a stupid toggle. I'm more inclined to just remove the toggle instead.

**Comment by Techjar** [comment]
What I could possibly do is remove the toggle from the SteamVR bindings and make it an in-game option instead, so that it can apply only to actually sneaking, and not exiting vehicles.
